cns/service/main.go

```diff
@@ -309,6 +326,11 @@ func main() {
 	// Create logging provider.
 	logger.InitLogger(name, logLevel, logTarget, logDirectory)
 
+	if clientDebugCmd != "" {
+		cnsclient.HandleCNSClientCommands(clientDebugCmd, clientDebugArg)
+		os.Exit(0)
```

Comment:
I think we should still return 1 when an error happened instead of always returning 0 even though this is just a debug tool